source/cgame/cg_cmds.cpp

```diff
@@ -939,11 +947,119 @@ void CG_GameCommand( const char *command )
 /*
 ==========================================================================
 
+BLOCKLIST
+
+==========================================================================
+*/
+
+blockentry_t blockentries[MAX_BLOCKS];
```

Comment:
Considering this is `c++`, why not use an `std::map<uint64_t, std::string>` or `std::unordered_map<uint64_t, std::string>` ?